src/static/localizations/en.yml

```diff
@@ -189,6 +189,15 @@ purchase:
     ethereum_insufficient_funds: Ethereum account has insufficient balance to perform this transaction
     listing_failed: Unable to list this NFT
 
+withdrawal:
```

Comment:
apologies if this is a distraction, not sure if you want these, can just revert the l10n, or can run through the whole WithdrawlModal file, or..